Makefile

```diff
@@ -249,8 +249,9 @@ clean-lib:
 			$(call clean-source-lib, $(source))))
 
 clean-runtime-helpers:
-	rm -rf packages/babel-runtime/helpers
-	rm -rf packages/babel-runtime-corejs2/helpers
+	rm -f packages/babel-runtime/helpers/**/*.js
+	rm -f packages/babel-runtime-corejs2/helpers/**/*.js
+	rm -f packages/babel-runtime-corejs3/helpers/**/*.js
 	rm -rf packages/babel-runtime-corejs2/core-js
```

Comment:
nit: I think we should also clean `packages/babel-runtime-corejs3/core-js(-stable)?`.

Reply:
Could that explain something like that appearing in the past hours?

```
Global error: Uncaught Error: Could not find module `@babel/runtime/helpers/esm/typeof.js` imported from `ember-exam/test-support/-private/ember-exam-qunit-test-loader` at http://localhost:7357/assets/vendor.js, line 252
```